certified-connectors/Oodrive Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2891 @@
+      "get": {
+        "summary": "Resend the notification email to all recipients",
+        "description": "Send a notification email to all the recipients of a specific contract",
+        "operationId": "ResendNotificationAllRecipients",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "ID of the contract to resend notification",
+            "x-ms-summary": "Contract ID",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "succeed": {
+                  "type": "boolean",
+                  "description": "Specifiy if the notification was successfuly resend"
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Contract not found",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "string",
+                  "description": "Error message"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/contracts/{id}/transaction/prooffiles": {
+      "get": {
+        "summary": "Download a contract's proof file",
+        "description": "Download the proof file of the contract",
+        "operationId": "DownloadProofFileContract",
+        "consumes": [
+          "application/zip"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "ID of the contract to download the proof file",
+            "x-ms-summary": "Contract ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "filename",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Name of the file which will be downloaded",
+            "x-ms-summary": "Output file name",
+            "default": "proof.zip"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "type": "string",
+            "default": "application/zip",
+            "x-ms-visibility": "internal",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "File downloaded",
+            "schema": {
+              "type": "file"
+            }
+          }
+        }
+      }
+    },
+    "/contracts/recipients/{cfc_id}": {
+      "delete": {
+        "summary": "Delete a recipient for contract",
+        "description": "Delete a recipient from a contract, by his recipient for contract ID. More information in the documentaton.",
```

Comment:
@QwerioTEAM all files should be free of grammatical errors. please change "documentaton" to "documentation"